lib/Casa.js

```diff
@@ -51,6 +51,12 @@ class Casa {
     this.config = cfg;
     const { config } = this;
 
+    // function middleware
+    // Additional middleware handler
+    if (!['undefined', 'function'].includes(typeof config.mountController)) {
```

Comment:
Might it also be worth checking if the function is already bound? This would avoid unexpected behaviour when we later call `config.mountController.call({ expressApp, expressRouter }, mountCommonMiddleware);`

Reply:
Good point, I've added an extra check for `.prototype` on the function:
https://github.com/dwp/govuk-casa/pull/7/files#diff-d10348d94a46c35c1126f167b163f269R58